### ISSUE TITLE
Failed to precompile NPZ

### ISSUE THREAD

**Issue by Proyag** [fhs/NPZ.jl#25]
Installed using `Pkg.add("NPZ")` on a fresh install of Julia 1.0.0. Got the following error when I tried to use `import NPZ`/`using NPZ`.

```
julia> import NPZ
[ Info: Precompiling NPZ [15e1cf62-19b3-5cfa-8e77-841668bca605]
ERROR: LoadError: syntax: extra token "Header" after end of expression
Stacktrace:
 [1] include at ./boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at ./loading.jl:1038
 [3] include(::Module, ::String) at ./sysimg.jl:29
 [4] top-level scope at none:2
 [5] eval at ./boot.jl:319 [inlined]
 [6] eval(::Expr) at ./client.jl:389
 [7] top-level scope at ./none:3
in expression starting at /home/pal/.julia/packages/NPZ/onMHH/src/NPZ.jl:140
ERROR: Failed to precompile NPZ [15e1cf62-19b3-5cfa-8e77-841668bca605] to /home/pal/.julia/compiled/v1.0/NPZ/TV7Kn.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:852
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:834
```

Output of `versioninfo()`, if it's helpful:
```
julia> versioninfo()
Julia Version 1.0.0
Commit 5d4eaca0c9 (2018-08-08 20:58 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)
```

**Comment by fhs** [comment]
This should be fixed once https://github.com/JuliaLang/METADATA.jl/pull/17211 is merged and you update to v0.3.0. Alternatively, use master for now.

**Comment by Proyag** [comment]
Thanks, works now.
